generators/add-web-assets/exc-react/templates/src/components/ActionsForm.js

```diff
@@ -160,15 +160,15 @@ const ActionsForm = (props) => {
     }
     try {
       // invoke backend action
-      const actionResponse = await actionWebInvoke(action, headers, params)
+      const actionResponse = await actionWebInvoke(actions[actionName], headers, params)
```

Comment:
IMO, UI components don't need to load the actions list
In case of `ActionForm` we load config, to list the actions
I think its more cleaner code to just invoke the action with name and data

Reply:
The way it is:
- The UI needs to load them to get the list to display in the dropdown.
- The UI calls util with the 'name' of an action, so util ALSO needs to lookup that name

This is simplified by having one or the other being the owner of the list.  Right now there is no clear owner, and a fragile shared dependency which is not clearly defined.